BREAKING_CHANGES_LOG.md

```diff
@@ -2,6 +2,15 @@ Before 1.0, the stack do NOT follow semver version in releases.
 
 This document aims to ease the WIP migration from a version to another by providing intels about what to do to migrate.
 
+## v0.129.0
+* Component: TreeView
+* PR: [feat(SelectObject): display results as ListGroupItem](https://github.com/Talend/ui/pull/880)
+* Changes :
+
+| Before | After |
+|---|---|
+| undefined-add | nothing |
```

Comment:
Hmm this is not understandable.
Please add some sentences to explain that the ids are not present anymore if no id is provided as props.